keras_cv/layers/preprocessing/random_brightness.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+_SCALE_VALIDATION_ERROR = (
+    "The `scale` should be number or a list of two numbers "
+    "that ranged between [-1.0, 1.0]. "
+)
+
+
+class RandomBrightness(tf.keras.layers.Layer):
+    """Randomly adjust brightness for the a RGB image.
+
+    This layer will randomly increase/reduce the brightness for the input RGB image.
+    During inference time, the output will be identical to input. Call the layer with
+    training=True to adjust brightness of the input.
+
+    Note that different brightness adjustment will be apply to each the images in the
+    batch.
+
+    Args:
+        scale: Float or a list/tuple of 2 floats between -1.0 and 1.0. The scale is
+            used to determine the lower bound and upper bound of the brightness
+            adjustment. A float value will be choose randomly between the limits.
+            When -1 is chosen, the output image will be black, and when 1.0 is
+            chosen, the image will be fully white. When only one float is provided,
+            eg, 0.2, then -0.2 will be used for lower bound and 0.2 will be used for
+             upper bound.
```

Comment:
Should we also support integers? Interpreted as a range in absolute levels, e.g. `32` would be interpreted as a +-32 shift of the channels